src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -1659,11 +1659,14 @@ internal SignatureHelp GetSignatureHelp(TextDocumentPosition textDocumentPositio
             this.currentCompletionParseInfo = scriptParseInfo;
             resultCompletionItems = result.CompletionItems;
 
-            // Expanding star expressions in query
-            CompletionItem[] starExpansionSuggestion = AutoCompleteHelper.ExpandSqlStarExpression(scriptDocumentInfo);
-            if (starExpansionSuggestion != null)
+            // Expanding star expressions in query if the script is connected to a database.
```

Comment:
Would probably be good to specifically say that the parser requires a connection to determine the column names